code/game/machinery/computer/abnormality_work.dm

```diff
@@ -13,6 +13,10 @@
 	var/meltdown_time = 0
 	/// Can the abnormality even meltdown?
 	var/can_meltdown = TRUE
+	/// Will the abnormality send it's information along?
+	var/ignore_ordeal = FALSE
```

Comment:
Absolutely not needed.

Reply:
Yes, it is actually needed, from what I gathered. The meltdown variable goes away when work is started and we need to send info after work is ended. If I move when the meltdown variable is set to after work then the counter ticks down during work